app/models/service_provider_request.rb

```diff
@@ -1,16 +1,27 @@
 class ServiceProviderRequest
   attr_accessor :uuid, :issuer, :url, :ial, :aal, :requested_attributes
 
```

Comment:
Also can we add a comment up top saying that any change to these parameters needs to be orchestrated carefully because they don't roll up OR DOWN easily? because this obj is serialized/deserialized in Redis

Reply:
(update I think we can wait for another PR for a comment like this, right now would like to focus on doing getting this rolled forward & fixed)